src/main/java/duke/Task.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+/**
+ * Represents a Task
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Class Constructor that creates a task based on the description.
+     *
+     * @param description
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
This is interesting to note. I'm personally fine with or without.
![image](https://user-images.githubusercontent.com/39460640/131535283-c8eb7217-d265-4f07-964b-982a07a77935.png)